lib/gtfsplus/actions/gtfsplus.js

```diff
@@ -173,107 +175,104 @@ export function uploadGtfsPlusFeed (feedVersionId: string, file: File) {
       return
     }
     return uploadFile({file, url, token: user.token})
-      .then(result => {
-        console.log(result)
-        return dispatch(uploadedGtfsPlusFeed())
-      })
+      .then(result => dispatch(uploadedGtfsPlusFeed()))
   }
 }
 
+type GTFSPlusEntity = {
```

Comment:
This is a kind of random placement.  I think it'd be a good convention to have all flow type definitions occur immediately after imports and just before the rest of the code.